src/js/game/components/item_processor.js

```diff
@@ -30,7 +31,7 @@ export class ItemProcessorComponent extends Component {
             nextOutputSlot: types.uint,
             type: types.enum(enumItemProcessorTypes),
             inputsPerCharge: types.uint,
-
+            chargeWhenBlocked: types.bool,
```

Comment:
This change may break save data. It may be worth considering reverting it and having the buffer use 0 for `inputsPerCharge`.